BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -199,7 +209,23 @@ public final class BlueprintView: UIView {
         /// views that are actually generated by the root element.
         return rootController.children
     }
-    
+
+    private var environment: Environment {
```

Comment:
Pretty nit-picky, but since this var is used to create a new environment when the view update happens, I think semantically it makes sense to make this a function (`makeCurrentEnvironment` or something). Wouldn't want someone to "optimize" this by making it a non-computed property.